docs/api.md

```diff
@@ -146,7 +146,7 @@ Represents a Modulino Thermo module.
 #### Methods
 
 - **`float getHumidity()`**  
-  Returns the humidity reading.
+  Returns the humidity reading as a percentage in the range 0 to 100.
```

Comment:
```suggestion
  Returns the relative humidity reading as a percentage in the range 0 to 100.
```

I think it is good to specify exactly what is being read (and thus unambiguously what the returned percentage value represents).